mqtt_node:
    ros__parameters:
      vehicle_id: 1
	  mqtt_addr: 20.126.131.210
